src/js/constants.js

```diff
@@ -11,8 +11,6 @@ export const MediaUIEvents = {
     MEDIA_ENTER_PIP_REQUEST: 'mediaenterpiprequest',
     MEDIA_EXIT_PIP_REQUEST: 'mediaexitpiprequest',
     MEDIA_PLAYBACK_RATE_REQUEST: 'mediaplaybackraterequest',
-    MEDIA_CHROME_ELEMENT_CONNECTED: 'mediachromeelementconnected',
```

Comment:
Sorry, I didn't mean I thought you should remove support completely, just from the UI elements so we were't registering them multiple ways.

However, we could still pull this into a separate PR. I think we might want to explore the [context proposal](https://github.com/webcomponents/community-protocols/blob/main/proposals/context.md) and see if there's any structure we want to align with. I at least like that the event names are framed as a 'request'.